DEVELOP.rst

```diff
@@ -122,3 +122,33 @@ The rpm package can be installed as follows:
 ::
 
     $ sudo yum install pgcli*.rpm
+
+Running the integration tests
+-----------------------------
+
+Integration tests use `behave package http://pythonhosted.org/behave/`_.
+Configuration settings for this package are provided via ``behave.ini`` file
+in root directory.
+
+The database user ``pg_test_user`` has to have permissions to create and drop
```

Comment:
The setting in .ini file is called "pg_test_user". I updated the readme to make that clear.